hvplot/plotting/core.py

```diff
@@ -45,31 +45,34 @@ class hvPlotBase:
     Examples
     --------
 
-    >>> import hvplot.pandas
-    >>> from bokeh.sampledata.degrees import data as deg
-
-    >>> line = deg.hvplot(
-    ...     x="Year",
-    ...     y=["Art and Performance", "Business", "Biology", "Education", "Computer Science"],
-    ...     value_label="% of Degrees Earned by Women",
-    ...     legend="top",
-    ...     height=500,
-    ...     width=620,
-    ...     kind="line",
-    ... )
-    >>> line
+    .. code-block::
```

Comment:
using .. code-block:: will enable better sphinx documentation and it renders just as well in VS Code. I changed this systematically because the code is easier to copy and use without the `>>>`. Less friction.